### DISCUSSION TITLE
App not installed

### DISCUSSION THREAD

**Discussion by qaywsxedcrfvtgb** [erap320/LenovoWatchMOD#26] [Q&A] [answered]
I am trying to install the app (3.0.3) but it fails and tells "App not installed." What to do?
Thanks
(Pocophone f1, global)

**Comment by erap320** [comment]
Con you make sure the original version has beed succesfully uninstalled from the Play Store before trying to install the modded APK?

**Comment by qaywsxedcrfvtgb** [comment]
It is. I am sure.
(Miui global 10.3.5 stable)

**Comment by erap320** [comment, marked as answer]
I'm not exactly sure what could be the cause of this error, but I'll try suggesting you two things:

- Try disabling Google Play Protect from the Play Store (some people on StackOverflow found this useful)
- Try clearing "old apps data" with "SD Maid", you can find it in the store. There may be other apps that do the same type of cleaning, but I only know this one

**Comment by qaywsxedcrfvtgb** [comment]
Disabling play protect did the job, thanks :) I am looking forward to try the app!

**Comment by erap320** [comment]
You're welcome! Hope you'll enjoy it ;)

**Comment by qaywsxedcrfvtgb** [comment]
Hi, I just wanted to tell you that it works just fine. Login to my account
worked and I managed to upgrade my watch's firmware to 0.4.0 finally.
Notification for gmail also worked.
Thanks! :)

čt 4. 7. 2019 v 23:04 odesílatel ERap320 <notifications@github.com> napsal:

**Comment by erap320** [comment]
I'm really glad to hear it! Thank you for contributing by pointing out the installation problem, I'm sure it will help other users with the same issue.
